pages/steps/permutations-utils.tsx

```diff
@@ -715,7 +698,7 @@ export const failedStepsWithRetryButtonInteractive: ReadonlyArray<StepsProps.Ste
     header: <span style={{ margin: '0' }}>Could not fetch security groups</span>,
     details: (
       <>
-        <span style={{ margin: '0' }}>The request expired due to a server issue.</span>
+        <span style={{ marginBlock: 0, fontSize: '12px' }}>The request expired due to a server issue.</span>
```

Comment:
Why font size and the margin block? Where is the margin coming from that we need to set it to 0? For font size maybe you can check Box and its settings